protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/enocean_config/enocean_config.txt

```diff
@@ -0,0 +1,8 @@
+#Wed Dec 17 17:11:53 CET 2014
+#0x00112233_RORG_FUNC_TYPE_FRIENDLYNAME_DESCRIPTION=aa#bb#cc##dd
```

Comment:
Well, I was also wondering about all other lines in this file as it refers to specific products from specific manufacturers. Why is this needed inside a generic base driver? If it is only for tests, it should at least not end up in the final jar.